src/main/java/io/mvnpm/esbuild/install/WebDepsInstaller.java

```diff
@@ -34,76 +29,82 @@ public final class WebDepsInstaller {
 
     private static final ObjectMapper mapper = new ObjectMapper();
 
-    public static boolean install(Path nodeModulesDir, List<WebDependency> dependencies) throws IOException {
```

Comment:
@edewit this boolean is used: https://github.com/quarkiverse/quarkus-web-bundler/blob/main/deployment/src/main/java/io/quarkiverse/web/bundler/deployment/WebBundlerProcessor.java#L192

Reply:
Right all you do there is test if the dependencies changed and only then call install, but the same logic also exist here, so it would be better to let this take care of it instead of duplicating this logic

Reply:
> Right all you do there is test if the dependencies changed and only then call install, but the same logic also exist here, so it would be better to let this take care of it instead of duplicating this logic

how to " test if the dependencies changed"